src/providers/ldap/sdap.c

```diff
@@ -923,7 +928,8 @@ errno_t setup_tls_config(struct dp_option *basic_opts)
         ret = ldap_set_option(NULL, LDAP_OPT_X_TLS_CERTFILE, tls_opt);
         if (ret != LDAP_OPT_SUCCESS) {
             DEBUG(SSSDBG_CRIT_FAILURE,
-                  "ldap_set_option failed: %s\n", sss_ldap_err2string(ret));
+                  "ldap_set_option failed(certfile): %s\n",
```

Comment:
"failed" misplaced, should be after function parameter:
"ldap_set_option(certfile) failed"